ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -62,6 +62,22 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-sts-shared-vpc-critical-high-f3
+  cron: 22 15 1,4,7,10,13,16,19,22,25,28 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-sv
+      ENABLE_SHARED_VPC: "yes"
+      QE_USAGE: rosa-sts-shared-vpc-critical-high-f3
+      REGION: us-east-1
```

Comment:
Is there any function enabled in the profile those cannot fit for current region?

Reply:
No.
Actually, `us-east-1` is more complicated  than other regions, see https://github.com/terraform-redhat/terraform-provider-rhcs/pull/569/files#diff-af47971dcdc8b7bd6bff3593ed96629cb6ce60b778725013948b3e9c5e133d0aR506-R510